autoapi/extension.py

```diff
@@ -67,6 +69,23 @@ def run_autoapi(app):
     )
     url_root = os.path.join("/", app.config.autoapi_root)
 
+    # check backend availability
+    # in reality if user will attempt to use `go` api type without
+    #  sphinxcontrib.golangdomain installed, sphinx will prohibit
+    #  all operations because of sphinxcontrib.golangdomain
+    #  extension is required
```

Comment:
I don't think this comment is needed. You have made the error message nice and clear, so it is obvious what the code is checking without needing a comment.